apache/airflow-ci-infra
# Fix runner AMI so it runs

- Update to the latest runner version
- Install the vector.toml config file
- Install stop-runner-if-no-job in to correct path
- Don't enable actions.runner service at boot (do it slightly later in user data)

Yes, this "should" be multiple PRs, but sometimes it's just overkill
